schema/CVE_Record_Format.json

```diff
@@ -500,6 +500,93 @@
         },
+        "applicability": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/applicabilityElement"
+            }
+        },
+        "applicabilityElement": {
+            "description": "Affected products defined using one or more software identifiers and applicability parameters. An operator property allows AND or OR logic between identifiers or combinations thereof. The negate and vulnerable Boolean properties allow applicability combinations to be inverted and/or defined as vulnerable or not. Multiple version fields are provided for capturing ranges of products when defining a Match String Range. NOTE: When defining an applicability block, it is recommended that it align with (as much as possible) the product data provided within the affected block.",
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "enum": ["AND", "OR"]
+                },
+                "negate": {
+                    "type": "boolean"
+                },
+                "nodes": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/node"
+                    }
+                }
+            }
+        },
+        "node": {
+            "description": "Defines a configuration node in an applicability statement.",
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "enum": ["AND", "OR"]
+                },
+                "negate": {
+                    "type": "boolean"
+                },
+                "cpeMatch": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/cpe_match"
+                    }
+                },
+                "purlMatch": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/purl_match"
+                    }
+                }
+            },
+            "allOf": [
+                { "required": ["operator"] },
+                {
+                    "anyOf": [
+                        { "required": ["cpeMatch"] },
+                        { "required": ["purlMatch"] }
+                    ]
+                }
+            ]
+        },
+        "purl_match": {
+            "description": "purl match string or range",
+            "type": "object",
+            "properties": {
+                "vulnerable": {
+                    "type": "boolean"
+                },
+                "criteria": {
+                    "description": "Placeholder until we find a formal purl schema",
+                    "$ref": "#/definitions/uriType"
+                },
+                "matchCriteriaId": {
+                    "$ref": "#/definitions/uuidType"
+                },
+                "versionStartExcluding": {
```

Comment:
For version ranges, purl uses this https://github.com/package-url/purl-spec/blob/main/VERSION-RANGE-SPEC.rst